yt/loaders.py

```diff
@@ -1291,6 +1296,11 @@ def load_sample(fn, progressbar: bool = True, timeout=None, **kwargs):
     the data registry, the ones passed to this function take priority.
     """
 
+    if fn is None:
+        print("One can see which sample datasets are available at: https://yt-project.org/data")
+        print("or alternatively by running: yt.sample_data.api.get_data_registry_table()")
+        return
```

Comment:
Here's an actual suggestion :)
```suggestion
    if fn is None:
        print(
            "One can see which sample datasets are available at: https://yt-project.org/data\n"
            "or alternatively by running: yt.sample_data.api.get_data_registry_table()",
            file=sys.stderr
        )
        return None
```

Reply:
to avoid dupplicating data between the message and the docstring, we can also do this:
```suggestion
    if fn is None:
        print(load_sample.__doc__, file=sys.stderr)
        return None
```